src/pages/certifications/front-end-libraries/react/pass-an-array-as-props/index.md

```diff
@@ -38,7 +38,7 @@ const List= (props) => {
 // ... same as above
 ```
 
-The ```javascript .join(", ") ``` is used to separate diffetent elements of the array and display then with a comma and a space in between.
+The `.join(", ")` method is used to take each element from within the array and join them into a string to be displayed.
 
 We are using the modularity of React in this example to display the tasks passed by two different components to a common componenet which renders the final HTML.
```

Comment:
Typo `componenet`, should be `component`.
Otherwise looks great 👍

Reply:
Thanks @ezioda004 I have corrected the typo now.